src/components/tx/TxHistory/List/transaction.js

```diff
@@ -22,9 +22,11 @@ export default connect(
 
     return {
       showFiat: launcher.selectors.getChainName(state).toLowerCase() === 'mainnet',
+      network: state.network.toJS(),
```

Comment:
This will cause re-rendering component because toJS() produces new object every time. Do we need whole network structure or currentBlockNumber is enough here ?